package.json

```diff
@@ -35,23 +35,22 @@
   "license": "MIT",
   "repository": "jest-community/jest-extended",
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
     "@types/jest": "^23.3.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
     "babel-eslint": "^8.0.1",
-    "babel-jest": "^23.0.1",
+    "babel-jest": "^24.0.0",
     "babel-jest-assertions": "^0.1.0",
     "babel-plugin-gwt": "^1.0.0",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "codecov": "^3.0.0",
+    "core-js": "^3.11.0",
     "eslint": "^4.9.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jest": "^21.2.0",
     "husky": "^0.14.3",
     "import-all.macro": "^2.0.3",
-    "jest": "^23.0.1",
+    "jest": "^24.0.0",
```

Comment:
This seems like a breaking change.

Reply:
That's a dev dependency, and master is already bumped to 24.